### PR TITLE
update oauth tests for azure

### PR DESCRIPTION
**Pull request by jmspring**

### CODE REVIEW

File: azurecompute-arm/pom.xml [hunk 1]
```diff
@@ -36,8 +36,8 @@
     <test.jclouds.azurecomputearm.subscription-id>FIXME_subscription-id</test.jclouds.azurecomputearm.subscription-id>
     <test.jclouds.azurecomputearm.tenant-id>FIXME_tenant-id</test.jclouds.azurecomputearm.tenant-id>
     <!-- note that there's no prefix of "test." in the url.  This is to ensure that jclouds inteprets the value, as opposed to maven -->
-    <test.azurecomputearm.endpoint>https://management.azure.com/subscriptions/${test.jclouds.azurecomputearm.subscription-id}</test.azurecomputearm.endpoint>
-    <test.azurecomputearm.api-version>2014-04-01</test.azurecomputearm.api-version>
+    <test.azurecomputearm.endpoint>https://management.core.windows.net/subscriptions/${test.jclouds.azurecomputearm.subscription-id}</test.azurecomputearm.endpoint>
```
Comment (ritazh) [inline azurecompute-arm/pom.xml:39]: I thought all azure arm rest api endpoints be `https://management.azure.com`. Is there a reason for using `https://management.core.windows.net`?

**Comment by ritazh** [comment]
@jtjk @isalento Code review needed before this can be merged into the `resourcegroupapi` branch.

**Comment by jtjk** [comment]
LGTM

**Comment by isalento** [comment]
:+1:

**Comment by jmspring** [comment]
Redoing against head.

**Pull request closed without merging**